### PR TITLE
Added example how to work with BigInt (js)

### PR DESCRIPTION
**Pull request by garikbesson**

### CODE REVIEW

**Comment by garikbesson** [comment]
@race-of-sloths

**Comment by race-of-sloths** [comment]
### 🙁 This repository is not a part of the [Race of Sloths](https://race-of-sloths.com)
Please ask the maintainer to [contact us](https://github.com/NEAR-DevHub/race-of-sloths/issues/new/choose) to join the race!

<details><summary><strong>What is the Race of Sloths</strong></summary>
<p>

Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag us inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/garikbesson)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!

</p>
</details>

File: collections-js/src/contract.ts [hunk 1]
```diff
@@ -31,6 +32,17 @@ class Storage {
     this.greeting = greeting;
   }
 
+  @view({}) // This method is read-only and can be called for free
+  get_big_int(): string {
```
Comment (gagdiez) [inline collections-js/src/contract.ts:36]:
```suggestion
  get_big_int(): bigint {
```

File: collections-js/src/contract.ts [hunk 2]
```diff
@@ -31,6 +32,17 @@ class Storage {
     this.greeting = greeting;
   }
 
+  @view({}) // This method is read-only and can be called for free
+  get_big_int(): bigint {
+    return this.big_int as unknown as string; // Because of serializing process it actually returns BigInt as string for view methods
```
Comment (gagdiez) [inline collections-js/src/contract.ts:37]:
```suggestion
    return this.big_int
```

**Comment by gagdiez** [comment]
@race-of-sloths score 3

**Pull request merged**